boot/zephyr/targets/nrf52840_pca10056.h

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (C) 2017, Linaro Ltd
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_TARGETS_NRF52840_PCA10056_
+#define H_TARGETS_NRF52840_PCA10056_
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
+#define FLASH_AREA_IMAGE_0_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_1_OFFSET	0x82000
+#define FLASH_AREA_IMAGE_1_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_SCRATCH_OFFSET	0xfc000
+#define FLASH_AREA_IMAGE_SCRATCH_SIZE	0x01000
```

Comment:
Performance isn't affected that much by the scratch size, however, you will be doing a lot of erases on the scratch area, which reduces the lifetime of the device.  With this configuration, each update will do 122 erases of the scratch sector (for one erase of each data sector). Given that some of the devices are only good for thousands of erases, this could result in a dead part fairly shortly. It might be worth trying to get out of Nordic what the expected lifetime should be.